mix.exs

```diff
@@ -66,7 +67,11 @@ defmodule Hexpm.MixProject do
       {:rollbax, "~> 0.5"},
       {:sweet_xml, "~> 0.5"},
       {:telemetry_poller, "~> 0.4"},
-      {:telemetry_metrics, "~> 0.4"}
+      {:telemetry_metrics, "~> 0.4"},
+      {:muzak,
+       git: "https://hexpm:sli_em-sl\{e91Ab@git.devonestes.com/muzak/muzak.git",
```

Comment:
@devonestes I think we _could_ load from there, but I am still not convinced it is the best practice. the bar for security is lower when a dep is only loaded into the test env, but I'd like to get some feedback from @ericmj and @wojtekmach about it as well.

Reply:
What about only loading it in CI? You could have something like:

if System.get_env("CI") do
  deps ++ [{:muzak, ...}]
else
  deps
end